src/timer.cpp

```diff
@@ -75,15 +76,15 @@ void Timer::UpdateMMIO() {
 
 u64 Timer::GetMaxSkip() const {
     if (pause || count_mode == CountMode::EventCount)
-        return Infinity;
+        return std::numeric_limits<u64>::max();
 
     if (counter == 0) {
         if (count_mode == CountMode::AutoRestart) {
             return ((u32)start_high << 16) | start_low;
         } else if (count_mode == CountMode::FreeRunning) {
             return 0xFFFFFFFF;
         } else /*Single*/ {
-            return Infinity;
+            return std::numeric_limits<u64>::max();
```

Comment:
The if right above this one should probably also use std::numeric_limits for uniformity